tf2/src/buffer_core.cpp

```diff
@@ -1296,11 +1354,16 @@ void BufferCore::_getFrameStrings(std::vector<std::string> & vec) const
   boost::mutex::scoped_lock lock(frame_mutex_);
 
   TransformStorage temp;
+  if (frameIDs_reverse.size() <= 1u)
+  {
+    return;
+  }
```

Comment:
not really because if the size is 1 (only NO_PARENT in the tree, I don't want to consider it.